File: Reverse.c
#include<stdio.h>
#include<string.h>
void main()
{
char a[100],n;
int i=0,j=0;
printf("Enter the string ");
gets(a);
while(i<j) {
temp = str[i];
str[i] = str[j];
str[j] = temp;
i++;
j--;
}
printf("\n Reverse string is :%s", str);
return (0);
}

File: Reverse.java
import java.util.Scanner;
public class Reverse{
public static void main(String args[])
{
int n,rev=0;
Scanner scan=new Scanner(System.in);
System.out.println("Enter the number");
n=scan.nextInt();
temp=n;
while(temp!=0)
{
rev=rev*10;
rev=n%10+rev;
n=n/10;
}
System.out.println("Reverse of a number"+rev);
}
}
